src/components/MovieShowPage.js

```diff
@@ -23,91 +23,134 @@ export const MovieShowPage = props => {
         const newRatings = { ...user, ratings: ratingData.ratings };
         updateUser(newRatings);
       });
-    });
+    })
   };
 
   const findUserRating = id => {
     const userRatings = props.user.ratings.map(rating => rating.movie_id);
     if (userRatings.includes(id)) {
       return props.user.ratings.find(movie => movie.movie_id === id).rating;
     } else {
-      return 'Please rate the movie below...';
+      return '...';
     }
   };
 
+  const handleDeleteRating = event => {
+    const { updateUser, user } = props;
+    deleteRating(findRatingId(id), user.id)
+      .then(data => {
+        fetchRatings(user.id).then(ratingData => {
+          const updatedRatings = { ...user, ratings: ratingData.ratings };
+          updateUser(updatedRatings);
+      });
+    })
+  };
+
+  const findRatingId = id => {
+    const movieIds = props.user.ratings.map(rating => rating.movie_id);
+    if (movieIds.includes(id)) {
+      return props.user.ratings.find(movie => movie.movie_id === id).id;
+    }
+    console.log(props.user.ratings.find(movie => movie.movie_id === id).id)
+  };
+
   return (
     <div className='movie-page'>
       <img className='backdrop' src={backdrop_path} alt={title} />
-      <h1>{title}</h1>
-      <p>In Theaters: {release_date}</p>
+      <h1 className='movie__title--two'>{title}</h1>
+      <p className='in-theaters'>In Theaters: {release_date}</p>
       <p className='overview'>{overview}</p>
-      <p>Average Rating: {average_rating}</p>
-      {isLoggedIn && <div>My Rating: {findUserRating(id)}</div>}
-      <div className='rating-bar'>
-        <p className='rating-description'>Rate Film:</p>
-        <div className='rating-scale'>
-          <button
-            className='rating-btn btn-one'
-            value='1'
-            onClick={event => handleRatingsUpdates(event)}>
-            1
-          </button>
-          <button
-            className='rating-btn btn-two'
-            value='2'
-            onClick={event => handleRatingsUpdates(event)}>
-            2
-          </button>
-          <button
-            className='rating-btn btn-three'
-            value='3'
-            onClick={event => handleRatingsUpdates(event)}>
-            3
-          </button>
+      <p className='average__rating--two'>Average Rating: {Math.round( average_rating * 10 ) / 10}</p>
+      {isLoggedIn &&  (findUserRating(id) < 10) &&
+        <div className='already-rated'>
+          <p className='my-rating'>
+            My Rating: {findUserRating(id)}
+          </p>
```

Comment:
If the only difference between the conditional rendering for `isLoggedIn &&  (findUserRating(id) < 10)`  and `isLoggedIn &&  (findUserRating(id) === 10)` is the className (lines 66 and 78), could we dry this up by declaring the classname with a ternary?  

Something like:
`findUserRating(id) === 10 ? 'top-marks' : 'my-rating'` ?

Reply:
Dry is best. I'll look up how to do this and get it changed. Much appreciated.